### PR TITLE
add python formatting proposal

### PR DESCRIPTION
**Pull request by dustinblack**
## Changes introduced with this PR

Adding a proposal to standardize on one python code formatting solution.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by portante — COMMENTED** [review]
Approve.

Not marking an official approval since I am a core member of the Pbench repository.

**Review by jaredoconnell — APPROVED** [review]
That seems fine to me.

**Review by jdowni000 — APPROVED** [review]

**Comment by jdowni000** [comment]
Seems fine to me

**Comment by portante** [comment]
@janosdebugs, see https://github.com/arcalot/arcalot-round-table/pull/18, I did not push it to that branch since it was @dustinblack's proposal.

**Review by portante — APPROVED** [review]

**Review by jdowni000 — APPROVED** [review]

File: art-decisions/proposals/2022-10-26-standardize-python-formatting.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Acceptance of this proposal would require applying the formatting first to all
+existing code at the same time as adding the `pre-commit` hooks.
+
+## Recommendation
+
+We should set one standard and adhere to it judiciously without exception. I
+recommend that we simply adopt the standards already established by the
+[pbench](https://github.com/distributed-system-analysis/pbench) repository and
+eliminate further debate on the topic. This formatting standard should also be
+applied wherever possible with automation and CI.
```
Comment (jaredoconnell) [inline art-decisions/proposals/2022-10-26-standardize-python-formatting.md:36]: Based on Janos's feedback, it may make sense to reword it to something like:
"It is important that we adhere to one standard judiciously and eliminate further debate on the topic. The standards documented above match the standards already adopted by the [pbench](https://github.com/distributed-system-analysis/pbench) repository. This formatting standard should also be applied wherever possible with automation and CI.

**Review by portante — APPROVED** [review]

**Review by mfleader — APPROVED** [review]

**Review by mkarg75 — APPROVED** [review]

File: art-decisions/proposals/2022-10-26-standardize-python-formatting.md (hunk 1 above)
Comment (sandrobonazzola) [reply art-decisions/proposals/2022-10-26-standardize-python-formatting.md:38]: Was this comment addressed?

**Review by sandrobonazzola — CHANGES_REQUESTED** [review]
please review @jaredoconnell comment

File: art-decisions/proposals/2022-10-26-standardize-python-formatting.md (hunk 1 above)
Comment (Harshith-umesh) [reply art-decisions/proposals/2022-10-26-standardize-python-formatting.md:38]: nope, not yet.

Comment (dustinblack) [reply art-decisions/proposals/2022-10-26-standardize-python-formatting.md]: update pushed

**Review by Harshith-umesh — APPROVED** [review]

**Review by sandrobonazzola — APPROVED** [review]
LGTM
